flink-tests/src/test/scala/org/apache/flink/api/scala/migration/StatefulJobSavepointMigrationITCase.scala

```diff
@@ -1,380 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.flink.api.scala.migration
-
-import org.apache.flink.FlinkVersion
-import org.apache.flink.api.common.accumulators.IntCounter
-import org.apache.flink.api.common.functions.{OpenContext, RichFlatMapFunction}
-import org.apache.flink.api.common.state.{ListState, ListStateDescriptor, ValueState, ValueStateDescriptor}
-import org.apache.flink.api.java.functions.KeySelector
-import org.apache.flink.api.java.tuple.Tuple2
-import org.apache.flink.api.scala._
-import org.apache.flink.api.scala.migration.CustomEnum.CustomEnum
-import org.apache.flink.configuration.Configuration
-import org.apache.flink.contrib.streaming.state.EmbeddedRocksDBStateBackend
-import org.apache.flink.runtime.state.{FunctionInitializationContext, FunctionSnapshotContext, StateBackendLoader}
-import org.apache.flink.runtime.state.hashmap.HashMapStateBackend
-import org.apache.flink.runtime.state.memory.MemoryStateBackend
-import org.apache.flink.streaming.api.TimeCharacteristic
-import org.apache.flink.streaming.api.checkpoint.CheckpointedFunction
-import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment
-import org.apache.flink.streaming.api.functions.sink.RichSinkFunction
-import org.apache.flink.streaming.api.functions.source.SourceFunction
-import org.apache.flink.streaming.api.watermark.Watermark
-import org.apache.flink.test.checkpointing.utils.SnapshotMigrationTestBase
-import org.apache.flink.test.checkpointing.utils.SnapshotMigrationTestBase.{ExecutionMode, SnapshotSpec, SnapshotType}
-import org.apache.flink.test.util.MigrationTest
-import org.apache.flink.test.util.MigrationTest.ParameterizedSnapshotsGenerator
-import org.apache.flink.util.Collector
-
-import org.junit.Test
-import org.junit.runner.RunWith
-import org.junit.runners.Parameterized
-
-import javax.annotation.Nullable
-
-import java.util
-import java.util.function.BiFunction
-import java.util.stream.Collectors
-
-import scala.util.{Failure, Try}
-
-object StatefulJobSavepointMigrationITCase {
```

Comment:
Can not remove tests from `migration` dir.

Reply:
This test case primarily focus on Scala case class state migration, while the corresponding Java version is `org.apache.flink.test.checkpointing.StatefulJobSnapshotMigrationITCase`. Therefore, I believe this test case can be deleted. What do you think?